packages/theme_tailor/lib/src/generator/theme_tailor_generator.dart

```diff
@@ -124,9 +140,24 @@ class ThemeTailorGenerator extends GeneratorForAnnotation<Tailor> {
   }
 
   List<String> _computeThemes(ConstantReader annotation) {
-    return List<String>.from(
-      annotation.read('themes').listValue.map((e) => e.toStringValue()),
-    );
+    List<String> annotationThemes;
+    if (!annotation.read('themes').isNull) {
+      annotationThemes = List<String>.from(
+        annotation.read('themes').listValue.map((e) => e.toStringValue()),
+      );
+    } else {
+      annotationThemes = [];
+    }
+
+    var pubspecThemes = (builderOptions.config['themes'] as YamlList).map((element) => element.toString()).toList();
```

Comment:
config is already parsed and it has type `Map<String, dynamic>` so it's simple json not `YamlList `

Reply:
Ive checked twice and it was YamlList